doc/advance_install.rst

```diff
@@ -0,0 +1,267 @@
+
+
+Create a Python3 virtualenv to install Pytitiont's dependencies:
+
+.. code-block:: bash
+
+  cd /srv/pytition/
+  sudo virtualenv -p python3 pytition_venv
+
+Clone Pytition git repository and checkout latest release:
+
+.. code-block:: bash
+
+  cd www
+  sudo git clone https://github.com/pytition/pytition
+  cd pytition
+  sudo git checkout $version
+
+Attribuer les bons propriétaires et les bons droits aux dossiers:
+.. code-block:: bash
+  sudo chown -R pytition-admin:www-data /srv/pytition
+  sudo chown orga1-user:www-data /srv/pytition/www/mediaroot/orga1
+  sudo chown orga2-user:www-data /srv/pytition/www/mediaroot/orga2
+  sudo chmod g+s /srv/pytition/www/static/
+
+Enter your virtualenv and install Pytition's dependencies:
+
+.. code-block:: bash
+  sudo su pytition-admin
+  source /srv/pytition/pytition_venv/bin/activate
+  (pytition_venv) $ pip3 install -r /srv/pytition/www/pytition/requirements.txt
+
+Créer les bases de données db-pytition-orga1, db-pytition-orga2, db-pytition-admin ainsi 
+que les utilisateurs associés db-user-orga1, db-user-orga2 et db-user-admin sur votre serveur MariaDB
+
+Pour chaque organisation, écrire le fichier /etc/pytition/{orga1,orga2,admin}/my.cnf
+Exemple de fichier pour orga1
+  [client]
+  host = your-data-base-server
+  database = db-pytition-orga1
+  user = db-user-orga1
+  password = YOUR_PASSWORD_HERE
+  default-character-set = utf8
+
+Pour l'admin, on pourra utiliser une base sqlite3 plutôt que de créer une nouvelle base sur les serveur mariaDB
+
+Pour chaque organisation, créer le fichier /etc/pytition/{orga1,orga2,admin}/config.py en copiant par exemple le fichier 
+/srv/pytition/www/pytition/config_example.py
+
+Les fichiers my.cnf et config.py doivent avoir les bonnes permissions et droits. Par exemple pour orga1:
+
+.. code-block:: bash
+  sudo chown orga1:pytition-admin /etc/pytition/orga1/{my.cnf,config.py}
+  sudo chmod u=rw,g=r,o=--- /etc/pytition/orga1/{my.cnf,config.py}
+
+Now you can edit your config file in `pytition/pytition/settings/config.py` according to :ref:`Configuration`.
+
+You **must** *at least* configure the settings described in the :ref:`Mandatory settings<mandatory settings>` section of the :ref:`Configuration` page.
+
+Those are:
+
+.. hlist::
+
+  * SECRET_KEY
+  * STATIC_URL
+  * STATIC_ROOT
+  * MEDIA_URL
+  * MEDIA_ROOT
+  * DATABASES
+  * ALLOWED_HOSTS
+
+Attention aux valeurs suivantes:
+STATIC_ROOT = "/srv/pytition/www/static"
+MEDIA_ROOT = "/srv/pytition/www/mediaroot/orga1 (pour le config.py de l'orga1)
+la configuratio de DATABASE doit bien pointer sur /etc/pytition/orga1/my.cnf 
+
+
+.. note:: Do not forget to put a correct path to the `my.cnf` MySQL credential file in your each config `DATABASES` setting.
+
+Initialiser Pytition ainsi que les bases de données. Vous devez être dans le virtualenv pour entrer les commandes suivantes:
```

Comment:
Initialize Pytition as well as its databases. You must be in the virtualenv while entering the following commands: